iroha/tests/genesis.json

```diff
@@ -0,0 +1,55 @@
+{
+  "txs": [
+    {
+      "isi": [
+      {
+        "Register": {
+          "object": {
+            "Peer": {
+              "id": {
+                "address": "localhost:1337",
+                "public_key": "ed207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
+              },
+              "address": "localhost:1337",
+              "domains": {},
+              "trusted_peers_ids": [],
+              "triggers": [],
+              "parameters": []
+            }
+          },
+          "destination_id": {
+            "DomainName": "global"
+          }
+        }
+      },
```

Comment:
Please remove Register Peer, it might break consensus if in genesis, it is an edge case I will need to check later.

Reply:
ok!